### PR TITLE
Auth tests

### PR DESCRIPTION
**Pull request by agtugchik**

### CODE REVIEW

File: src/__tests__/index.test.js [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+
+  it('Should work input validation', async () => {
+    await waitFor(() => {
+      expect(getByText(signInTitle)).toBeInTheDocument();
+    });
+
+    const signInButton = getByText(signInButtonTitle).closest('button');
+    const emailInput = getByLabelText(emailInputLabel);
+    const passwordInput = getByLabelText(passwordInputLabel);
+    fireEvent.change(emailInput, { target: { value: notValidEmail } });
```
Comment (alexlatypov) [inline src/__tests__/index.test.js]: use userEvent instead of fireEvent

**Review by alexlatypov — CHANGES_REQUESTED** [review]

File: src/__tests__/index.test.js (hunk 1 above)
Comment (agtugchik) [reply src/__tests__/index.test.js]: fixed

File: src/components/ui/InputValidationError/InputValidationError.test.tsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  InputValidationError,
+} from 'components/ui/InputValidationError/InputValidationError';
+import { FormFieldsNames } from 'constants/form-fields-names';
+
+const { getByText } = screen;
+
+describe('InputValidationError component', () => {
+  afterEach(cleanup);
+
+  it('should throw email error', () => {
```
Comment (alexlatypov) [inline src/components/ui/InputValidationError/InputValidationError.test.tsx]: I think u can combine all `it` to one

**Review by alexlatypov — CHANGES_REQUESTED** [review]

File: src/components/ui/InputValidationError/InputValidationError.test.tsx (hunk 2 above)
Comment (agtugchik) [reply src/components/ui/InputValidationError/InputValidationError.test.tsx]: fixed

**Pull request merged**